test/runtests.jl

```diff
@@ -58,12 +55,18 @@ end
             @safetestset "Constants Test" include("constants.jl")
             @safetestset "Parameter Dependency Test" include("parameter_dependencies.jl")
             @safetestset "Generate Custom Function Test" include("generate_custom_function.jl")
-            @safetestset "Initial Values Test" include("initial_values.jl")
             @safetestset "Equation Type Accessors Test" include("equation_type_accessors.jl")
             @safetestset "Equations with complex values" include("complex.jl")
         end
     end
 
+    if GROUP == "All" || GROUP == "InterfaceI" || GROUP == "Initialization"
```

Comment:
```suggestion
    if GROUP == "All" || GROUP == "Initialization"
```

And just make a new test group in the Github CI. We keep putting everything into InterfaceI so fully splitting is good!